makefiles/Makefile.devhub-content

```diff
@@ -11,7 +11,6 @@ PRODUCTION_URL=https://developer.mongodb.com
 
 PROJECT=devhub
 MUT_PREFIX ?= $(PROJECT)
-PREFIX=devhub
```

Comment:
PREFIX is never referenced in this makefile (only MUT_PREFIX is) thus why I am removing it!